.github/workflows/build-check-install.yaml

```diff
@@ -146,28 +146,28 @@ on:
         default: ""
       skip-desc-branch:
         description: |
-          Passed to `insightsengineering/setup-r-dependencies`.
+          Passed to `r-lib/setup-r-dependencies`.
           Used only if deps-installation-method == 'setup-r-dependencies'.
         required: false
         type: boolean
         default: false
```

Comment:
this whole section is to be removed - `r-lib/setup-r-dependencies` does not take such parameter

same for other parts